provision/acc_provision/acc_provision.py

```diff
@@ -176,6 +176,11 @@ def config_default():
                 "name": "snat-operator",
                 "watch_namespace": "",
                 "globalinfo_name": "snatglobalinfo",
+                "portRange": {
+                    "start": 5000,
+                    "end": 65000,
+                    "portsPerNode": 3000,
```

Comment:
Hate to nit pick, but I think we should be consistent with the naming convention. Can we change portsPerNode to ports_per_node?
Also, can we change portRange to snat_port_range?

Reply:
Yes sure! I just reused the names I used in snat-operator code but here it makes more sense to use underscores.